data/avro-schema/src/main/resources/avro/net/corda/data/p2p/app/AuthenticatedMessageHeader.avsc

```diff
@@ -1,31 +1,37 @@
 {
   "type": "record",
   "name": "AuthenticatedMessageHeader",
+  "doc": "A message that will be delivered by the p2p layer over an end-to-end (authenticated and optionally encrypted) session.",
```

Comment:
We should probably do the same for `UnauthenticatedMessage`, `AuthenticatedMessage` and `AppMessage`.

Reply:
oh my bad, that was supposed to be in the corresponding messages - not the headers. I will move it.